test/library/standard/Path/prachi-muskaan/testcases_normPath.good

```diff
@@ -0,0 +1,14 @@
+foo/bar
+foo/bar
+foo/bar
+foo/bar
+/bar
+/foo/baz/bar
+foo/bar
+foo/bar
+foo
+/
+..
+.
+
```

Comment:
Line 13 and 14 won't actually get inserted in the output, so this test will fail when I try to run it